wiring/inc/spark_wiring_usbserial.h

```diff
@@ -46,7 +46,7 @@ class USBSerial : public Stream
     bool isEnabled();
     bool isConnected();
 
-	void begin(long speed);
+	void begin(long speed = 9600);
```

Comment:
I guess the rationale is that the baudrate doesn't actually affect the transfer, and is only "virtual" in the linerate descriptor?